static/css/responsive.css

```
@@ -205,6 +205,27 @@
 }
 
 /* ── High contrast mode ─── */
+@media (prefers-contrast: more) {
+    * {
+        border-color: currentColor !important;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)